lib/stylesheet/manager/builder.rb

```diff
@@ -86,19 +90,19 @@ def source_map_fullpath
   end
 
   def source_map_filename
-    "#{stylesheet_filename}.map"
+    "#{stylesheet_filename}.map?__ws=#{current_hostname}"
   end
 
   def stylesheet_fullpath_no_digest
     "#{cache_fullpath}/#{stylesheet_filename_no_digest}"
   end
 
-  def stylesheet_cdnpath(hostname)
-    "#{GlobalSetting.cdn_url}#{stylesheet_relpath}?__ws=#{hostname}"
+  def stylesheet_cdnpath
```

Comment:
I do not think this is used anywhere, but the next method. Does it make sense to keep it around?

Reply:
Removed :+1: